src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -578,4 +578,29 @@ button.mapboxgl-ctrl-zoom-out  {
 canvas.mapboxgl-canvas:focus {
   outline: 2px solid $color-gold-light;
   outline-offset: 2px;
+}
+
+.input-clear {
+  position: relative;
+  font-size: 16px;
+
+  @media only screen and (min-width: 980px) {
+    display: inline-block;
+  }
+}
+.input-clear .fa-clear {
```

Comment:
Please add a newline separator, thanks

Reply:
Also, since this is SASS, I think you can nest `.fa-clear` under `.input-clear` like so:

```sass
.input-clear {
...
  .fa-clear {
    ...
}
```